src/api/app/assets/javascripts/webui2/packages.js

```diff
@@ -34,3 +34,11 @@ $(function ($) {
     details.attr('open', null);
```

Comment:
I wonder why this file is called `packages.js` (in plural) and not `package.js`? :thinking: Any clue @openSUSE/open-build-service ? :thinking:

Reply:
@ChrisBr could answer that, but why we have ```package``` and ```packages```? Maybe in the past it used to be all ```packages``` ?

Reply:
Why do you think this matters?

Reply:
@hennevogel 

> Why do you think this matters?

consistency? I just didn't know where to put it until I found out that this was for package related things.. there is also a `package-view_file.js` (in singular). It confuses me... :roll_eyes:

Reply:
None of this is related to anything. Its one huge javascript file that get's included on every page. However you structure it internally is up to you :smile:

Reply:
I think we need some criteria for JavaScript file names, before this is a mess... :bulb: